documentation/examples/todoist/todoist.ts

```diff
@@ -1,100 +1,404 @@
+    },
+    favorite: {
+      description: "Is the project a favorite.",
+      type: coda.ValueType.Boolean,
+    },
+    projectId: {
+      description: "The ID of the project.",
+      type: coda.ValueType.Number,
+      required: true,
+    },
+    parentProjectId: {
+      description: "For sub-projects, the ID of the parent project.",
+      type: coda.ValueType.Number,
+    },
+    // Add a reference to the sync'ed row of the parent project.
+    // References only work in sync tables.
+    parentProject: ProjectReferenceSchema,
+  },
+  primary: "name",
+  id: "projectId",
+  featured: ["url"],
+  identity: {
+    name: "Project",
+  },
+});
+
+// A reference to a synced Task. Usually you can use
+// `coda.makeReferenceSchemaFromObjectSchema` to generate these from the primary
+// schema, but that doesn't work in this case since a Project itself can contain
+// a reference to a parent project.
+const TaskReferenceSchema = coda.makeObjectSchema({
+  type: coda.ValueType.Object,
+  codaType: coda.ValueHintType.Reference,
+  properties: {
+    name: {type: coda.ValueType.String, required: true},
+    taskId: {type: coda.ValueType.Number, required: true},
+  },
+  primary: "name",
+  id: "taskId",
+  identity: {
+    name: "Task",
+  },
+});
+
+const TaskSchema = coda.makeObjectSchema({
+  type: coda.ValueType.Object,
+  properties: {
+    name: {
+      description: "The name of the task.",
+      type: coda.ValueType.String,
+      required: true,
+    },
+    description: {
+      description: "A detailed description of the task.",
+      type: coda.ValueType.String,
+    },
+    url: {
+      description: "A link to the task in the Todoist app.",
+      type: coda.ValueType.String,
+      codaType: coda.ValueHintType.Url
+    },
+    order: {
+      description: "The position of the task in the project or parent task.",
+      type: coda.ValueType.Number,
+    },
+    priority: {
+      description: "The priority of the task.",
+      type: coda.ValueType.String,
+    },
+    taskId: {
+      description: "The ID of the task.",
+      type: coda.ValueType.Number,
+      required: true,
+    },
+    projectId: {
+      description: "The ID of the project that the task belongs to.",
+      type: coda.ValueType.Number,
+    },
+    parentTaskId: {
+      description: "For sub-tasks, the ID of the parent task it belongs to.",
+      type: coda.ValueType.Number,
+    },
+    // A reference to the sync'ed row of the project.
+    // References only work in sync tables.
+    project: ProjectReferenceSchema,
+    // Add a reference to the sync'ed row of the parent task.
+    // References only work in sync tables.
+    parentTask: TaskReferenceSchema,
+  },
+  primary: "name",
+  id: "taskId",
+  featured: ["project", "url"],
+  identity: {
+    name: "Task",
+  },
+});
+
+/**
+ * Convert a Project API response to a Project schema.
+ */
+function toProjectSchema(project: any, withReferences=false) {
+  let result: any = {
```

Comment:
Can maybe avoid the `: any`, or does TS complain on line 191 since you're assigning into an inferred literal type?